[Workers] Added note about "wrangler dev" websocket support in using-websockets.md (#1362)
cloudflare/cloudflare-docs: products/workers/src/content/learning/using-websockets.md

## Writing a WebSocket Server

<Aside>

> __Note:__ The `wrangler dev` tool currently does not support connecting to workers via the WebSocket protocol. For more information, you can [read the github issue here.](https://github.com/cloudflare/wrangler/issues/1910).

Reviewer: ```suggestion
__Note:__ The `wrangler dev` tool currently does not support connecting to Workers via the WebSocket protocol. Support for WebSocket support in `wrangler dev` is tracked in [this GitHub issue](https://github.com/cloudflare/wrangler/issues/1910).
```

Author: Thank you for the suggestion! I like the re-wording, so I've committed it. I appreciate the help.
